models/fates/R/met2model.FATES.R

```diff
@@ -32,23 +34,27 @@ met2model.FATES <- function(in.path, in.prefix, outfolder, start_date, end_date,
   # defining temporal dimension needs to be figured out. If we configure FATES to use same tstep then we may not need to change dimensions  
   
   
-  insert <- function(ncout, name, unit, data) {
-    var   <- ncdf4::ncvar_def(name = name, units = unit, dim = dim, missval = -6999, verbose = verbose)
-    ncout <- ncdf4::ncvar_add(nc = ncout, v = var, verbose = verbose)
+  insert <- function(ncout, name, unit, data, dim) {
+    var  <- ncdf4::ncvar_def(name, unit, dim = dim, missval = as.numeric(1.0e36), verbose = verbose)
+    ncout <- ncdf4::ncvar_add(ncout, var)
     ncvar_put(nc = ncout, varid = name, vals = data)
```

Comment:
Please call all your functions from a namespace
```suggestion
    ncdf4::ncvar_put(nc = ncout, varid = name, vals = data)
```